{
    "type": "bundle",
    "id": "bundle--0e1b0f3b-a39b-4027-ad32-8d095f61a79b",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--482d171e-cec2-4f56-93ce-1665c088fc6a",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-05-12T00:19:16.469916Z",
            "modified": "2021-05-12T00:19:16.469916Z",
            "name": "CVE-2021-30504",
            "description": "In JetBrains IntelliJ IDEA before 2021.1, DoS was possible because of unbounded resource allocation.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2021-30504"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}